app/src/main/java/org/oppia/app/splash/SplashFragment.kt

```diff
@@ -0,0 +1,69 @@
+package com.example.myapplication
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import androidx.fragment.app.Fragment
+import org.oppia.app.HomeActivity
+import org.oppia.app.R
+import android.app.Activity
+import android.content.Context
+import org.oppia.app.utility.Logger
+
+
+class SplashFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+
+
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        // Inflate the layout for this fragment
+        val view = inflater.inflate(R.layout.fragment_splash, container, false)
+
+        loadSplashScreen()
+        return view
+    }
+
+    private fun loadSplashScreen() {
+
+
+        Logger.d(requireContext(),"Splash","debug")
+
+        // TODO(Veena): Replace below code to load profile data and then redirect to HomePage
+
+        Handler().postDelayed(// Using handler with postDelayed called runnable run method
```

Comment:
I'm still concerned with this--we shouldn't be handling configuration changes for this activity, and removing that line may result in a crash if the user rotates during the transition.

Why do we need this wait? Can't we just immediately transition to the home activity?

Reply:
if we don't add wait time, transition is too quick. Will that be ok.? As it is setting background theme in the splashActivity. Splash screen will be shown until app resources are initialized which is hardly 1 to 2 seconds

Reply:
I have removed handler and now there is transition without delay